inspire/base/static/js/inspire_base_init.js

```diff
@@ -25,7 +25,8 @@ require([
   "js/feedback_inspire",
   "js/header/header",
   "js/notification",
-  "js/dropdown-menu/dropdown-menu"
+  "js/dropdown-menu/dropdown-menu",
+  "js/plots"
```

Comment:
@jmartinm We only need to load plots.js in the detailed record, I have a feeling there's another way to do it, other than here.